CRM/Utils/ICalendar.php

```diff
@@ -245,7 +237,7 @@ public static function createCalendarFile($info) {
           return strtotime($event['end_date'] ?? $event['start_date']);
         }, $info)
       );
-      $template->assign('timezones', CRM_Utils_ICalendar::generate_timezones($timezones, $date_min, $date_max));
+      $template->assign('timezones', CRM_Utils_ICalendar::generate_timezones($date_min, $date_max));
```

Comment:
Hrm. I may have clicked "Merge" too quickly. There's a signature change here, and I see an extension referencing it:

```
ext/eventics/eventics.php:    $template->assign('timezones', CRM_Utils_ICalendar::generate_timezones($timezones, $date_min, $date_max));
```